### PR TITLE
Task for adding untriaged bugs as new tasks

### PR DESCRIPTION
**Pull request by mbacovsky**
- new task to add untriaged bugs
- BZ query fully configurable in profile
- added api_key auth to BZ calls
- all BZ calls are using rest api
- added general ability to search BZ
- fixed setting of task complexity

### CODE REVIEW

**Comment by ares** [comment]
Looks reasonable, could you please update https://github.com/ares/kansync/blob/master/profiles/default.yml.example with the bz api config (IIUC) and also explain, how to get such token?

**Comment by mbacovsky** [comment]
@ares updated (in separate commit  that should be squashed)

**Comment by mbacovsky** [comment]
@ares, added cleanup of existing triage tasks. Now it seems it maintains the triage tasks properly.

File: profiles/default.yml.example [hunk 1]
```diff
@@ -64,3 +71,9 @@ configuration:
       # never move task to backlog automatically, once it's already in progress
       Backlog:
         column: ['Work in progress']
+  600-sync_bz_bugs_to_triage:
```
Comment (ares) [inline profiles/default.yml.example:74]: this should be 400 instead of 600

**Review by ares — APPROVED** [review]
Thanks, works great, I'll update the only minor thing just after merging!

**Pull request merged**